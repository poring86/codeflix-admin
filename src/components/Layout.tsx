import { Box, Container } from '@mui/material'
import type { ReactNode } from 'react'

type Props = {
  children: ReactNode
}

export function Layout({ children }: Props) {
  return (
    <Box>
      <Container maxWidth="lg" sx={{ mt: 4, mb: 4, color: 'white' }}>
        {children}
      </Container>
    </Box>
  )
}
