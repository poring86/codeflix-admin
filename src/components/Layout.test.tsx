import { render } from "@testing-library/react"
import Layout from "./Layout"

describe("Header", () => {
  test("should render header", () => {
    const { asFragment } = render(
      <Layout>
        test
      </Layout>
    )

    expect(asFragment()).toMatchSnapshot()
  })
})