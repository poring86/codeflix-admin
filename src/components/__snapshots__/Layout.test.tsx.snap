// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render header 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-0"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1xd9xi2-MuiContainer-root"
    >
      test
    </div>
  </div>
</DocumentFragment>
`;
